src/math/floorf.rs

```diff
@@ -1,10 +1,13 @@
 use core::f32;
+#[cfg(test)]
+use libm_test::nearest;
 
 /// Floor (f64)
 ///
 /// Finds the nearest integer less than or equal to `x`.
 #[inline]
 #[cfg_attr(all(test, assert_no_panic), no_panic::no_panic)]
+#[cfg_attr(test, nearest)]
```

Comment:
All unary functions taking 32-bit arguments can just be tested against all inputs (all 32-bit bit patterns). I don't know why would it make sense to special case any tests here.